openbas-front/src/admin/components/common/injects/InjectsListButtons.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { FunctionComponent, useContext } from 'react';
+import { ToggleButton, ToggleButtonGroup, Tooltip } from '@mui/material';
+import { BarChartOutlined, ReorderOutlined } from '@mui/icons-material';
+import { makeStyles } from '@mui/styles';
+import type { InjectOutputType } from '../../../../actions/injects/Inject';
+import { exportData } from '../../../../utils/Environment';
+import { useFormatter } from '../../../../components/i18n';
+import { InjectContext } from '../Context';
+import ImportUploaderInjectFromXls from './ImportUploaderInjectFromXls';
+import useExportToXLS from '../../../../utils/hooks/useExportToXLS';
+import { useHelper } from '../../../../store';
+import type { TagHelper } from '../../../../actions/helper';
+import ButtonPopover from '../../../../components/common/ButtonPopover';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+    gap: 10,
+  },
+}));
+
+interface Props {
+  injects: InjectOutputType[];
+  setViewMode?: (mode: string) => void;
+  showTimeline: boolean;
+  handleShowTimeline: () => void;
+}
+
+const InjectsListButtons: FunctionComponent<Props> = ({
+  injects,
+  setViewMode,
+  showTimeline,
+  handleShowTimeline,
+}) => {
+  // Standard hooks
+  const classes = useStyles();
+  const { t } = useFormatter();
+  const injectContext = useContext(InjectContext);
+
+  // Fetching data
+  const { tagsMap } = useHelper((helper: TagHelper) => ({
+    tagsMap: helper.getTagsMap(),
+  }));
+
+  const isAtLeastOneValidInject = injects.some((inject) => inject.inject_injector_contract?.injector_contract_content_parsed !== null);
```

Comment:
Can you do this here ? !inject.inject_injector_contract?.injector_contract_content_parsed instead of !== null ? To manage undefined or empty value too ?

Reply:
Good point
I made the change